pkgs/development/compilers/ghcjs-ng/8.6/stage0.nix

```diff
@@ -59,20 +59,21 @@
     }) {};
 
   ghc-api-ghcjs = callPackage
-    ({ mkDerivation, array, base, binary, bytestring, containers
+    ({ mkDerivation, alex, array, base, binary, bytestring, containers
     , deepseq, directory, filepath, ghc-boot, ghc-boot-th, ghc-heap
-    , ghci-ghcjs, hpc, process, stdenv, template-haskell-ghcjs
+    , ghci-ghcjs, happy, hpc, process, stdenv, template-haskell-ghcjs
     , terminfo, time, transformers, unix
     }:
     mkDerivation {
       pname = "ghc-api-ghcjs";
-      version = "8.6.2";
+      version = "8.6.5";
       src = configuredSrc + /lib/ghc-api-ghcjs;
       libraryHaskellDepends = [
         array base binary bytestring containers deepseq directory filepath
         ghc-boot ghc-boot-th ghc-heap ghci-ghcjs hpc process
         template-haskell-ghcjs terminfo time transformers unix
       ];
+      libraryToolDepends = [ alex happy ];
```

Comment:
Did you change this manually, or did `genStage0` do it?

Reply:
This file comes from `genStage0`.